src/mixins/translatable.js

```diff
@@ -35,6 +36,20 @@ export default {
     listeners () {
       window.addEventListener('scroll', this.translate, false)
       window.addEventListener('resize', this.translate, false)
+
+      let isFirefox = window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1
+      let isEdge = window.navigator.userAgent.toLowerCase().indexOf('edge') > -1
+
+      if (isFirefox || isEdge) requestAnimationFrame(this.checkScrolling)
+    },
+
+    checkScrolling () {
+      if (this.lastScroll !== window.pageYOffset) {
+        this.lastScroll = window.pageYOffset
+        this.translate()
+      }
+
+      requestAnimationFrame(this.checkScrolling)
```

Comment:
This will *slaughter* performance. Checking pageYOffset on every frame even if nothing's happening is a terrible idea. It will also leak loads of memory because it keeps looping even after the component is destroyed.